### ISSUE TITLE
Dockerfiles for lucid, precise, and saucy

### ISSUE THREAD

**Issue by stuartpb** [stuartpb/rethinkdb-dockerfiles#10]
Upstream provides packages for each of these:

- http://download.rethinkdb.com/apt/pool/lucid/main/r/rethinkdb/
- http://download.rethinkdb.com/apt/pool/precise/main/r/rethinkdb/
- http://download.rethinkdb.com/apt/pool/saucy/main/r/rethinkdb/

It wouldn't be too tough to make Dockerfiles to install each of these, but since there's no tags for these versions in the official registry, it's kind of a moot point.

**Comment by stuartpb** [comment]
Actually, Precise does have an upstream tag: https://registry.hub.docker.com/_/ubuntu/

**Comment by gabor-boros** [comment]
@stuartpb can I close this issue? I believe this is not needed anymore since those distros have no support.

**Comment by stuartpb** [comment]
IMO, if a distro isn't supported, then it shouldn't be hosted under https://download.rethinkdb.com/apt/pool/

**Comment by gabor-boros** [comment]
@stuartpb you are right. @srh what do you think if we remove the unsupported distros from the download server?

**Comment by srh** [comment]
Why would we do that?

**Comment by stuartpb** [comment]
IMO, if a distro isn't supported, then it shouldn't be hosted under https://download.rethinkdb.com/apt/pool/

**Comment by srh** [comment]
Why?

**Comment by gabor-boros** [comment]
@srh, @stuartpb I had some time to think about this. As I can see, we should host the older binaries somewhere anyway as an "archive". For this repository, I would only keep those dockerfiles which are supported. Since the download server contains every package, anyone can have a dockerfile for an older RethinkDB version.

To conclude this, what if we:
- removing the older dockerfiles from here (to keep this repo clean)
- keeping all the packages on the download server
- closing this issue

What are your opinion?

**Comment by srh** [comment]
I'm not a user of the docker files, so I don't have any opinion on that.
